index.html

```diff
@@ -59,8 +52,14 @@
   <div class="line"><br /></div> <!-- end #line -->
 </div><!-- end #line-block -->
 
-<!-- iviewer release going out later>
-<p><img src="images/indexNew530.png" alt="New flag image" class="flag-left"/><a href="web-client.html" target="blank">OMERO.iviewer</a> - the Full Viewer in OMERO.web has been completely revamped as a WebApp using new technologies, allowing you to draw ROIs in OMERO.web, and much more. For further details see the <a href="web-client.html">Using OMERO.web</a> page.</p> -->
+<p>
+    <img src="images/indexNew530.png" alt="New flag image" 
+    class="flag-left"/><a href="web-client.html" 
+    target="blank">OMERO.iviewer</a> - the Full Viewer in OMERO.web has been 
+    completely revamped as a WebApp using new technologies, allowing you to 
+    draw ROIs in OMERO.web, and much more. For further details see the <a 
+    href="web-client.html">Using OMERO.web</a> page.
```

Comment:
Do we need this? It's just the same link as the one earlier in the line.

Reply:
If we keep it probably don't need "the ... page", the link itself suffices as "... see *Using OMERO.web*"